cpp/src/io/parquet/reader_impl_helpers.cpp

```diff
@@ -161,7 +180,8 @@ type_id to_type_id(SchemaElement const& schema,
   switch (physical) {
     case BOOLEAN: return type_id::BOOL8;
     case INT32: return type_id::INT32;
-    case INT64: return type_id::INT64;
+    case INT64:
+      return (duration_type.id() == cudf::type_id::EMPTY) ? type_id::INT64 : duration_type.id();
```

Comment:
```suggestion
      return (duration_type.id() == type_id::EMPTY) ? type_id::INT64 : duration_type.id();
```
let's remain consistent

Reply:
🙏